### ISSUE TITLE
Unable to install NVDA; Setup process halts abruptly

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4843]
**Reported by Sam_M on 2015-01-23 20:57**
I am attempting to install the latest version of NonVisual Desktop Access (NVDA) application onto my computer, but I am encountering some technical issues with the installation.

I accessed the download link from the following page : http://sourceforge.net/projects/nvda/files/

When I attempted to download the executable using the Chrome browser, the file was saved onto my computer. Double-clicking the executable and selecting 'Run" on the "Security Warning" window, starts the setup process by displaying a window with the message: "Please wait while Setup is loading". 

But the setup process ends abruptly (the window disappears and the music halts). The Setup Wizard never appears on my desktop. 

I also attempted to download the executable using the Internet Explorer browser. Instead of saving the file onto my device, I selected "Run" when prompted whether I wanted to save or run the file. Once the file downloaded, I was presented with the "Security Warning" window, and selected run. I was then presented with another error message that states: "Windows cannot access the specified device, path, or file. You may not have the appropriate permissions to access the item".

My local technical support team has tried diagnosing the cause of this problem, but unfortunately, have not been able to successfully install the application. My machine has been granted full permissions, so I should not be encountering any issues with the setup process.

I have also attempted to install a previous release of NVDA (2014.3) and experienced the same setup failure.

Is there a work-around for this issue that will enable me to install the application onto my machine?

Thank You!

**Comment by ehollig** [comment]
Closing, as we unfortunately are not able to contact the original poster for more information on this issue. Also, this indeed sounds like an antivirus blocking NVDA in sum way, but not exactly sure as we would need more information.
